MaterialDesignThemes.Wpf/PopupBox.cs

```diff
@@ -394,7 +400,9 @@ public override void OnApplyTemplate()
             _popup = GetTemplateChild(PopupPartName) as PopupEx;
             _popupContentControl = GetTemplateChild(PopupContentControlPartName) as ContentControl;
             _toggleButton = GetTemplateChild(TogglePartName) as ToggleButton;
-            
+
+            _popup.CommandBindings.Add(new CommandBinding(ClosePopupCommand, ClosePopupHandler));
```

Comment:
I would add a null check here because it is possible that `_popup` will be `null`.
```suggestion
            _popup?.CommandBindings.Add(new CommandBinding(ClosePopupCommand, ClosePopupHandler));
```